dev-util/vint/vint-0.3.21.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```

Comment:
Looking at upstream, they seem to support python3.7 and 3.8 already. Could you check whether they work or not?

Reply:
They both appear to work, but tests fail for 3.8, so I just added 3.7. I'll look into the test failure later.

Also, I added 3.7 and 3.8 to dev-python/ansicolor's python compat in this PR for this purpose.

Reply:
I went ahead and added 3.8 too. The failures are due to new 3.8 warnings that show up in stderr only during tests due to disabled byte code compilation in src_test. Aside from the unexpected stderr output, the tests function as expected. https://github.com/Vimjas/vint/issues/355